src/web/lib/storage.js

```diff
@@ -4,76 +4,92 @@ import { normalizeTheme, themesEqual } from "../../lib/themes";
 
 const log = makeLog("web.storage");
 
-const THEME_STORAGE_PREFIX = "THEME-";
-
-const themeStorageKey = key => `${THEME_STORAGE_PREFIX}${key}`;
-
-const isThemeStorageKey = key => key.indexOf(THEME_STORAGE_PREFIX) === 0;
-
-const themeKeyFromStorage = storageKey =>
-  storageKey.substr(THEME_STORAGE_PREFIX.length);
-
-const generateThemeKey = () =>
-  `${Date.now()}-${Math.floor(Math.random() * 1000)}`;
-
 let currentSavedTheme;
 
-function putTheme(key, theme) {
-  let isThemeDuplicate = checkDuplicateTheme(theme);
-  if (!isThemeDuplicate) {
-    log("putTheme", key, theme);
-    const storageKey = themeStorageKey(key);
-    localStorage.setItem(
-      storageKey,
-      JSON.stringify({
-        theme,
-        modified: Date.now()
-      })
-    );
-    currentSavedTheme = theme;
-    notifySelfForStorage(storageKey);
+class Storage {
```

Comment:
Refactoring here to make the localStorage handling generally usable between themes and images. Could have found a library to use here, but meh.